src/main/java/org/subethamail/smtp/helper/BasicMessageListener.java

```diff
@@ -18,6 +20,6 @@
      * @throws RejectException
      *             when caller to be given an SMTP error response
      */
-    void messageArrived(MessageContext context, String from, String to, byte[] data) throws RejectException;
+    void messageArrived(MessageContext context, String from, List<String> to, byte[] data) throws RejectException;
```

Comment:
FYI I'll look at possibly adding a single-to-field default method to the interface that delegates to this one after merge.